app/forms.py

```diff
@@ -90,6 +90,16 @@ class Meta:  #pylint: disable=no-init,missing-docstring,old-style-class,too-few-
         choices=lambda: [(v, lazy_gettext(v)) for v in current_app.config['DOMAINS']])
 
 
+class SharedMessageForm(Form):
+    '''
+    Form for submittin a message to share with the network.
+    '''
+
+    message = TextAreaField(
+        label=lazy_gettext('Have a question or something to share about innovation?')
```

Comment:
Not entirely sure if a `<label>` will be the right place to put this text from the wireframes, but it should make it accessible to screen readers at least...